docs/backup-restore-integration.md

```diff
@@ -0,0 +1,349 @@
+- ("", "Pod", "ns1", "virt-launcher-vmi1-XXXXX") \*
+
+\* Each VirtualMachineInstance has a corresponding uniquely named Pod.  The backup process can look up the name of this pod by using the `kubevirt.io/created-by=<UID of VirtualMachineInstance>` label selector.
+
+#### spec.priorityClassName
+
+```yaml
+...
+spec:
+  priorityClassName: pc1
+...
+```
+
+- ("scheduling.k8s.io", "PriorityClass", "", "pc1")
+
+#### spec.volumes[\*].persistentVolumeClaim
+
+```yaml
+...
+spec:
+  volumes:
+  - name: v1
+    persistentVolumeClaim:
+      claimName: pvc1
+...
+```
+
+- ("", "PersistentVolumeClaim", "ns1", "pvc1")
+- ("cdi.kubevirt.io", "DataVolume", "ns1", "pvc1")
+
+#### spec.volumes[\*].dataVolume
+
+```yaml
+...
+spec:
+  volumes:
+  - name: v1
+    dataVolume:
+      name: dv1
+...
+```
+
+- ("", "PersistentVolumeClaim", "ns1", "dv1")
+- ("cdi.kubevirt.io", "DataVolume", "ns1", "dv1")
+
+#### spec.volumes[\*].configMap
+
+```yaml
+...
+spec:
+  volumes:
+  - name: v1
+    configMap:
+      name: cm1
+...
+```
+
+- ("", "ConfigMap", "ns1", "cm1")
+
+#### spec.volumes[\*].secret
+
+```yaml
+...
+spec:
+  volumes:
+  - name: v1
+    secret:
+      name: s1
+...
+```
+
+- ("", "Secret", "ns1", "s1")
+
+#### spec.volumes[\*].serviceAccount
+
+```yaml
+...
+spec:
+  volumes:
+  - name: v1
+    serviceAccount:
+      serviceAccountName: sa1
+...
+```
+
+- ("", "ServiceAccount", "ns1", "sa1")
+
+#### spec.volumes[\*].memoryDump
+
+```yaml
+...
+spec:
+  volumes:
+  - name: v1
+    memoryDump:
+      claimName: pvc1
+...
+```
+
+- ("", "PersistentVolumeClaim", "ns1", "pvc1")
+
+#### spec.networks[\*].multus
+
+```yaml
+...
+spec:
+  networks:
+  - name: ovs-net
+    multus:
+      networkName: ovs-vlan-100
+...
+```
+
+- ("k8s.cni.cncf.io", "NetworkAttachmentDefinition", "ns1", "ovs-vlan-100")
+
+#### spec.accessCredentials[\*].sshPublicKey
+
+```yaml
+...
+spec:
+  accessCredentials:
+  - sshPublicKey:
+      source:
+        secret:
+          secretName: my-pub-key
+...
+```
+
+- ("", "Secret", "ns1", "my-pub-key")
+
+#### spec.accessCredentials[\*].userPassword
+
+```yaml
+...
+spec:
+  accessCredentials:
+  - userPassword:
+      source:
+        secret:
+          secretName: my-user-password
+...
+```
+
+- ("", "Secret", "ns1", "my-user-password")
+
+### VirtualMachineInstanceReplicaSet Object Graph
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstanceReplicaSet
+metadata:
+  name: vmirs1
+  namespace: ns1
+...
+```
+
+- ("kubevirt.io", "VirtualMachineInstanceReplicaSet", "ns1", "vmirs1")
+- ("kubevirt.io", "VirtualMachineInstance", "ns1", "vmirs1XXXX1") \*
+- ("kubevirt.io", "VirtualMachineInstance", "ns1", "vmirs1XXXX2") \*
+
+\* there are usually multiple VirtualMachineInstances corresponding to a VirtualMachineInstanceReplicaSet.  The backup process can look up the name of this pod by using the `kubevirt.io/vmReplicaSe=<name of VirtualMachineInstanceReplicaSet>` label selector.
```

Comment:
`kubevirt.io/vmReplicaSe=<name of VirtualMachineInstanceReplicaSet>`
I believe you are missing 't' at the end of `vmReplicaSe`